internal/blueprint/blueprint_test.go

```diff
@@ -89,3 +90,22 @@ func TestBumpVersion(t *testing.T) {
 		}
 	}
 }
+
+func TestGetPackages(t *testing.T) {
+
+	bp := Blueprint{
+		Name:        "packages-test",
+		Description: "Testing GetPackages function",
+		Version:     "0.0.1",
+		Packages: []Package{
+			{Name: "tmux", Version: "1.2"}},
+		Modules: []Package{
+			{Name: "openssh-server", Version: "8.*"}},
+		Groups: []Group{
+			{Name: "anaconda-tools"}},
+	}
+	out := bp.GetPackages()
+	assert.Equal(t, out[0], "tmux-1.2")
+	assert.Equal(t, out[1], "openssh-server-8.*")
+	assert.Equal(t, out[2], "@anaconda-tools")
```

Comment:
The arguments definition for assert functions show that expected value if first, then actual value is second. The order will not affect the assertion but will affect the error message in case of failure.

You may also try `assert.ElementsMatch` which will allow you to compare the arrays directly.

Reply:
Great idea, looks much better this way, thanks.